src/routers/alpha-router/alpha-router.ts

```diff
@@ -351,7 +351,11 @@ export type AlphaRouterConfig = {
   /**
    * Flag that allow us to override the cache mode.
    */
-  overwriteCacheMode?: CacheMode
+  overwriteCacheMode?: CacheMode;
+  /**
+   * Flag for token properties provider to enable fetching fee-on-transfer tokens.
+   */
+  enableFeeOnTransferFeeFetching?: boolean;
```

Comment:
this is not used anywhre right?

Reply:
What is ryoshi role in here ?

Reply:
This is used in `TokenPropertiesProvider`, where the flag has to be true to fetch the token fees. Otherwise, the provider returns empty map.

Reply:
Does that mean Ryoshi is a validator?